packages/suite/src/views/wallet/receive/components/CoinjoinReceiveWarning.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { darken } from 'polished';
+import { useDispatch, useSelector } from '@suite-hooks';
+import { FiatValue, FormattedCryptoAmount, Translation } from '@suite-components';
+import { Button, Icon, variables, Warning } from '@trezor/components';
+import { hideCoinjoinReceiveWarning } from '@suite-actions/suiteActions';
+import { formatAmount, getAccountDecimals } from '@suite-common/wallet-utils';
+import { UNECONOMICAL_COINJOIN_THRESHOLD } from '@suite/services/coinjoin';
+import { selectSelectedAccount } from '@wallet-reducers/selectedAccountReducer';
+import { selectIsAccountWithRatesByKey } from '@suite-common/wallet-core';
+
+const StyledWarning = styled(Warning)`
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    ${variables.SCREEN_QUERY.MOBILE} {
+        display: grid;
+        grid-template-columns: 1fr auto;
```

Comment:
Not sure what is the point of `grid-template-columns` here. Anyway, maybe a simpler solution (would not require lines 67-70):
```suggestion
        align-items: stretch;
        flex-direction: column;
        gap: 16px;
```

Reply:
0f15f72eb